pkg/releaseutil/manifest.go

```diff
@@ -37,12 +38,12 @@ func SplitManifests(bigfile string) map[string]string {
 	// Basically, we're quickly splitting a stream of YAML documents into an
 	// array of YAML docs. In the current implementation, the file name is just
 	// a place holder, and doesn't have any further meaning.
-	sep := "\n---\n"
+	sep := regexp.MustCompile("(?:^|\\s*\n)---\\s*")
```

Comment:
- [ ] Consider declaring this as a top-level `var`.  
  Doing so avoids compiling the regular expression on every invocation, though I don't know how often this function will be called.